packages/platforms/celestica/x86-64/x86-64-cel-redstone-xp/onlp/builds/src/module/src/i2c_dev.h

```diff
@@ -0,0 +1,158 @@
+#ifndef I2C_DEV_HH
```

Comment:
This file is already available for all userspace implementations. 
#include <linux/i2c-devices.h>

Please do not redefine.